### DISCUSSION TITLE
How to enable webassembly

### DISCUSSION THREAD

**Discussion by calvinbui** [Eloston/ungoogled-chromium#2408] [Q&A] [answered]
I've updated from 112 to 114 and sites using webassembly are no longer working. I find this to be an problem on Bromite as well, if a patch was taken from there.

Examples:
- https://liona24.github.io/wasm-tetris/
- https://www.blazorfluentui.net/
- https://privatebin.net/

I've tried enabling all the chrome flags for webassembly, but that didn't solve the problem.

**Comment by PF4Public** [comment, marked as answer]
We do not provide support for third-party binaries and I would not recommend downloading and installing them.
If you're seeking support file this issue with the repository you've downloaded the binary from.

**Comment by calvinbui** [comment]
thanks for the reply and insight

i was using [chrlauncher](https://github.com/henrypp/chrlauncher) which pulls from github.com/macchrome/winchrome. I always assumed it pulled directly from here. there is an [issue](https://github.com/macchrome/winchrome/issues/104) raised there about this problem.

**Comment by Rlbunsted** [comment]
> I've updated from 112 to 114 and sites using webassembly are no longer working. I find this to be an problem on Bromite as well, if a patch was taken from there.
> 
> Examples:
> 
> * https://liona24.github.io/wasm-tetris/
> * https://www.blazorfluentui.net/
> * https://privatebin.net/
> 
> I've tried enabling all the chrome flags for webassembly, but that didn't solve the problem.

Calvin I’m try to reach you I wonder if you wouldn’t be able to mod a Nintendo switch oled for me. Please email me rlbunsted@gmail.com I’m 100% real and would be willing to pay you

**Reply by Rlbunsted** [reply]
@calvinbui
